db/src/main/kotlin/edu/artic/db/models/ArticTour.kt

```diff
@@ -93,10 +93,10 @@ data class ArticTour(
     }
 
     /**
-     * Returns [floor], parsed to an integer. We default to [Int.MIN_VALUE] as 0 is a valid floor.
+     * Returns [floor], parsed to an integer. We default to 0 since some tours can be on multiple floors.
      */
     val floorAsInt: Int
-        get() = floor?.toIntOrNull() ?: Int.MIN_VALUE
+        get() = floor?.toIntOrNull() ?: 0
```

Comment:
Here we can't use 0. If the floor is empty iOS app assigns first stop's floor to the tour.

Reply:
noted. will change

Reply:
removed